web-console/webpack.config.js

```diff
@@ -124,10 +124,8 @@ module.exports = env => {
       ],
     },
     performance: {
-      hints: false
+      hints: false,
     },
-    plugins: [
-      // new BundleAnalyzerPlugin()
-    ],
+    plugins: process.env.BUNDLE_ANALYZER_PLUGIN === 'TRUE' ? [new BundleAnalyzerPlugin()] : [],
```

Comment:
@vogievetsky is this what you wanted? I documented it in both the build script and the read me

Reply:
looks good to me